documentation/basics/headless.md

```diff
@@ -8,74 +8,73 @@ weight: 6
 
-Before you can use headless mode, create a project and train a classifier.  Then save your project and quit ilastik.
+Except for the Carving Workflow, all workflows should be able to run in headless mode.
 
-To start ilastik in "headless" mode, use the `--headless` command-line flag.
+In order to run ilastik in headless mode, you will need to use the graphical user interface to create a project and train a classifier by manually drawing annotations as usual (see, e.g. [Pixel Classification Workflow]({{site.baseurl}}/documentation/pixelclassification/pixelclassification.html) or [Object Classification Workflow] for instructions on how to train your classifier). Once you're done with the training, save your project and quit ilastik.
 
-For example, on Linux:
+When running ilastik in headless mode, you must specify at least:
 
-    $ cd ilastik-1.1.7-Linux
-    $ ./run_ilastik.sh --headless
+- The `--headless` flag, to trigger headless operation;
+- Your pre-trained ilastik `.ilp` project file, via the `--project` argument (e.g. `--project=MyPixelClassProj.ilp`);
+- One or more files to be processed as a batch. Those should always be given as the last arguments to the command line.
 
-On Mac:
+For example, this command will run your classifier over 2 additional images (here we assume your classifier was trained on 2D images):
 
-    $ ./ilastik-1.1.7-OSX.app/Contents/ilastik-release/run_ilastik.sh --headless
+Linux:
 
-And on Windows:
+    $ cd ilastik-1.3.2-Linux/
+    $ ./run_ilastik.sh --headless --project=MyProject.ilp my_next_image1.png my_next_image2.png
 
-    $ cd "\Program Files\ilastik-1.0"
-    $ .\ilastik.bat --headless
+Mac:
 
-...but those commands will just start ilastik and immediately quit.
-To do something useful, you'll need to load your project file, and provide some input data for batch predictions.
-The following examples use linux shell syntax, but the options are the same for all platforms.
+    $ ./ilastik-1.3.2-OSX.app/Contents/ilastik-release/run_ilastik.sh  --headless --project=MyProject.ilp my_next_image1.png my_next_image2.png
 
-For example this example command will run your classifier over 2 additional images. 
-(Here we assume your classifier was trained on 2D images.)
+Windows:
 
-    $ ./run_ilastik.sh --headless --project=MyProject.ilp my_next_image.png my_next_image2.png
+    $ cd "\Program Files\ilastik-1.3.2"
+    $ .\ilastik.bat --headless --project=MyProject.ilp my_next_image1.png my_next_image2.png
 
-## Using stack input
+**Note:** the following examples use linux shell syntax, but the options are the same for all platforms.
+
+### Important Notes:
+
+- Except for options passed with the `--export_source` argument (see below), don't specify any values with space-characters in them, even if inside quotes (e.g. `"my file name.h5"`, or `"(10, 20)"`). Due to a [bug in cpython](https://bugs.python.org/issue22433) those might be misinterpreted.
```

Comment:
Could you please test if it works with an export source containing a space and and image as a positional argument? I would expect that you'd have to specify the type of data as an optional argument with the image in order to get it to work.